elyra/metadata/schemas/runtime-image.json

```diff
@@ -59,6 +59,14 @@
             "field_type": "dropdown",
             "category": "Source"
           }
+        },
+        "pull_secret": {
+          "title": "Image Pull Secret",
+          "description": "If specified, the value is used as container image registry key",
+          "type": "string",
+          "uihints": {
+            "category": "Source"
```

Comment:
Should we include the UI hint: `"secure": true,` here so that the UI masks the characters?  I think the names of secrets are meant to be treated as relatively secure items themselves.